docs/pbc/all/search/202212.0/third-party-integrations/integrate-algolia.md

```diff
@@ -1,44 +1,636 @@
+* `spryker/product-extension: "^1.4.0"`
+* `spryker/product-image: "^3.13.0"`
+* `spryker/product-label "^3.5.0"`
+* `spryker/product-label-storage "^2.6.0"`
+* `spryker/product-review: "^2.9.0"`
+* `spryker/search: "^8.19.3"`
+* `spryker/search-http: "^1.0.0"`
+* `spryker/store: "^1.17.0"`
+* `spryker/merchant-product-offer: "^1.5.0"` (Marketplace only)
+* `spryker/merchant-product-offer-data-import: "^1.1.0"` (Marketplace only)
+* `spryker/merchant-product-offer-search: "^1.4.0"` (Marketplace only)
+* `spryker/price-product-offer-data-import: "^0.7.1"` (Marketplace only)
+* `spryker/product-offer: "^1.4.0"` (Marketplace only)
 
-The Algolia app is now integrated to your store and starts exporting your product data automatically. Wait for a few minutes and go back to the Algolia website, to the **Overview -> Search -> Index** page. 
 
-{% info_block infoBox "Info" %}
+## Integrate Algolia
 
-The more products you have, the longer you have to wait until Algolia finishes the export. The average export speed is around 100 products per minute.
+Algolia integration is not a single step. Follow the steps in the sections below to integrate Algolia.
 
-{% endinfo_block %}
+### Shared configs
 
-The index is now populated with data from your store:
-![algolia-index-data](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/search/algolia/integrate-algolia/algolia-index-data.png)
+Add next config to `config/Shared/common/config_default.php`:
 
-For details on the created index data, see [Indexes](/docs/pbc/all/search/{{page.version}}/third-party-integrations/algolia.html#indexes).
\ No newline at end of file
+```php
+//...
+
+use Generated\Shared\Transfer\InitializeProductExportTransfer;
+use Generated\Shared\Transfer\ProductCreatedTransfer;
+use Generated\Shared\Transfer\ProductDeletedTransfer;
+use Generated\Shared\Transfer\ProductExportedTransfer;
+use Generated\Shared\Transfer\ProductUpdatedTransfer;
+use Generated\Shared\Transfer\SearchEndpointAvailableTransfer;
+use Generated\Shared\Transfer\SearchEndpointRemovedTransfer;
+
+//...
+
+$config[MessageBrokerConstants::MESSAGE_TO_CHANNEL_MAP] = [
+    //...
+    ProductExportedTransfer::class => 'product',
+    ProductCreatedTransfer::class => 'product',
+    ProductUpdatedTransfer::class => 'product',
+    ProductDeletedTransfer::class => 'product',
+    InitializeProductExportTransfer::class => 'product',
+    SearchEndpointAvailableTransfer::class => 'search',
+    SearchEndpointRemovedTransfer::class => 'search',
+];
+
+$config[MessageBrokerConstants::CHANNEL_TO_TRANSPORT_MAP] =
+$config[MessageBrokerAwsConstants::CHANNEL_TO_RECEIVER_TRANSPORT_MAP] = [
+    //...
+    'product' => MessageBrokerAwsConfig::SQS_TRANSPORT,
+    'search' => MessageBrokerAwsConfig::SQS_TRANSPORT,
+];
+
+$config[MessageBrokerAwsConstants::CHANNEL_TO_SENDER_TRANSPORT_MAP] = [
+    //...
+    'product' => 'http',
+    'search' => 'http',
+];
+```
+
+### Modules configuration and dependencies
+
```

Comment:
We can't have two titles without intro sentences. Add an introductory sentence that describes what this section is about and adjust the title in the way I mentioned earlier in the comments. Otherwise, remove the title.